Networking/Networking/Model/Customer.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 
 /// Represent a Customer Entity.
 ///
-public struct Customer: Decodable, Equatable {
+public struct Customer: Codable, Equatable {
```

Comment:
The `Codable` conformance here is missing, we are sending everything into the API request, included `siteID` and `userID` (which will be generated on the web).

Reply:
Added in 4a07d63.